MIGRATION.md

```diff
@@ -2,6 +2,19 @@
 
 The guides below are provided to ease the transition of existing applications using the Onfido SDK from one version to another that introduces breaking API changes.
 
+## `6.12.0` -> `6.14.0`
```

Comment:
Shouldn't this be `6.13.0`?
```suggestion
## `6.13.0` -> `6.14.0`
```

Reply:
I'm actually not sure what the convention is, but was following on from the last version listed in the doc.

Reply:
I think it's supposed to list the difference between the current version and the previous